### PR TITLE
Add "host" as parameter to all performance test results

### PR DESCRIPTION
**Pull request by jonmv**
@aressem please review. 
@baldersheim FYI.

### CODE REVIEW

File: lib/performance_test.rb [hunk 1]
```diff
@@ -216,6 +216,7 @@ def write_report(fillers)
     fillers.each do |filler|
       filler.call(@rep_result)
     end
+    @rep_result.add_parameter('host', `hostname`)
```
Comment (aressem) [inline lib/performance_test.rb:219]: This code is executed on the test runner / controller. Probably need to get the hostnames from the VespaModel and we can have more than one host.

**Review by aressem — CHANGES_REQUESTED** [review]

File: lib/performance_test.rb (hunk 1 above)
Comment (jonmv) [reply lib/performance_test.rb:219]: Hmm ... need to change performance data query to use "matches" to support multiple values of a parameter, but can do.

Comment (jonmv) [reply lib/performance_test.rb:219]: Meeeh, that may break other usages, though.

Comment (jonmv) [reply lib/performance_test.rb:219]: @baldersheim what tests really use several nodes?

Comment (jonmv) [reply lib/performance_test.rb:219]: Nvm, can make it work without breaking the others.

**Review by aressem — APPROVED** [review]

**Pull request merged**